examples/internal/proto/examplepb/a_bit_of_everything.pb.gw.go

```diff
@@ -491,6 +491,7 @@ func request_ABitOfEverythingService_CreateBody_0(ctx context.Context, marshaler
 	if berr != nil {
 		return nil, metadata, status.Errorf(codes.InvalidArgument, "%v", berr)
 	}
+
```

Comment:
While I think this is probably a bit nicer looking, I think we should try not to introduce a change like this that will affect all users upgrading their generator. It doesn't look trivial but it'd be nice if we can get it working!

Reply:
@johanbrandhorst I edited the template to check for `""` and avoid printing these unnecessary newlines.

Decided to do it directly inside the template as opposed to tweaking the output of the Prep functions.